src/smk/tool/search/tool-search-location.js

```diff
@@ -42,26 +42,26 @@ include.module( 'tool-search.tool-search-location-js', [
                 }
             } )
 
-            smk.on( this.id, {
-                'directions': function () {
-                    smk.$tool.directions.active = true
+            // smk.on( this.id, {
+            //     'directions': function () {
+            //         smk.$tool.directions.active = true
 
-                    smk.$tool.directions.activating
-                        .then( function () {
-                            return smk.$tool.directions.startAtCurrentLocation()
-                        } )
-                        .then( function () {
-                            return SMK.UTIL.findNearestSite( { latitude: self.feature.geometry.coordinates[ 1 ], longitude: self.feature.geometry.coordinates[ 0 ] } )
-                                .then( function ( site ) {
-                                    return smk.$tool.directions.addWaypoint( site )
-                                } )
-                                .catch( function ( err ) {
-                                    console.warn( err )
-                                    return smk.$tool.directions.addWaypoint()
-                                } )
-                        } )
-                }
-            } )
+            //         smk.$tool.directions.activating
+            //             .then( function () {
+            //                 return smk.$tool.directions.startAtCurrentLocation()
```

Comment:
I spent a while trying to update references to "directions" with the new tools (it was replaced by three - DirectionsOptionsTool, DirectionsRouteTool, and DirectionsWaypointsTool, the last seeming to be the most appropriate) but I wasn't able to get it working.